package sukkiri17;

import java.io.IOException;

public class Rensyu17_4 {

	public static void main(String[] args) throws IOException {//例外処理をキャッチせずに上に投げる
		throw new IOException("例外エラー");//例外を投げる

	}

}
